cwatm/hydrological_modules/water_quality/waterquality_main.py

```diff
@@ -45,9 +46,10 @@ def __init__(self, model):
         
         # define water quality sub-modules
         self.waterquality_p = waterquality_phosphorus(self)
+        self.erosed = waterquality_erosed(model)
```

Comment:
Here again about erosed - I would try to think on something a bit more intuitive to improve code readability. 
Maybe sediment?